templates/components/_nav_mobile.html.twig

```diff
@@ -1,22 +1,28 @@
 <nav class="gtm__navigation__mobile l__wrapper l__hide--above-small-laptop space--20-0">
     <ul class="list__links list__links--row list__links--nav l__row--h-stretch">
         <li><a href="{{ path('homepage') }}" class="link--no-decor text--logo">En Marche !</a></li>
+        <li><a class="btn btn--pink" href="{{ path('donation_index') }}">Donner</a></li>
     </ul>
     <div class="nav-mobile">
         <input type="checkbox" id="nav-menu">
         <label for="nav-menu">Menu de navigation</label>
         <div class="nav-mobile--menu">
-            {% if not app.user %}
                 <ul>
```

Comment:
extra indentation, since you remove the `{% if %}` block

Reply:
Done